lms/static/scripts/frontend_apps/components/GradingComment.tsx

```diff
@@ -65,11 +60,9 @@ function CommentPopover({
           Add a comment:
         </label>
         <div className="grow" />
-        <IconButton
+        <CloseButton
```

Comment:
The `CloseButton` has an implicit and consistent UI, and inherits the close callback from the `Dialog` via context.